src/kajal/WhileNumbers.java

```diff
@@ -0,0 +1,58 @@
+package kajal;
+
+public class WhileNumbers {
+	
+	int num1;
+	int num2;
+
+	void evenNumber(int start,  int end) {
+		num1 = start;
```

Comment:
you don’t require to create extra variables, you can handle this without it, think on this.

please refer below code
https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/208/files